get_linux/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<head>
+<title>Get QField for Windows</title>
```

Comment:
:penguin: :gun:

Reply:
I will keep it "Get QField" only.

Reply:
Or make it "Get QField for ${queryParams.nickname}".

Reply:
+1 to not name the platform.